doc/verification.rst

```diff
@@ -0,0 +1,106 @@
+Overview
+--------
+
+This is a SV/UVM testbench for the ibex core verification. This
+testbench loads the instruction binary generated from open source random
+instruction generator `riscv-dv`_, runs the RTL simulation, and compares
+the instruction trace against ISS simulation.
```

Comment:
We typically go for one sentence per line in this documentation.

Reply:
Done